File: MODUL-3/c/PRAK304-2310817210007-RaymondHariyono.c
#include <stdio.h>

int main(){
    int a;

for (int i = 0; i < 5; i++){
    scanf("%d", & a);
    if (a >= 100){
        printf("melebihi bilangan \n");}

    else if (a <= 9 && a >= 1){
        printf("satuan \n");
        }
     else if (a <= 19 && a >= 10){
        printf("belasan \n");
        }
    else if (a <= 99 && a >= 20){ 
        printf("puluhan \n");
        }
    else {printf("NOl \n");}
}
return 0;
}

File: MODUL-3/python/PRAK304-2310817210007-RaymondHariyono.py
for i in range(5):
    bilangan = int(input())

    if bilangan >= 1 and bilangan <= 10:
        print("Satuan")
    elif bilangan >= 11 and bilangan <= 19:
        print("belasan")
    elif bilangan >= 20 and bilangan <= 99:
        print("Puluhan")
    elif bilangan > 99:
        print("Anda Menginput Melebihi Limit Bilangan")
    elif bilangan == 0:
        print("Nol")
